src/test/java/net/openhft/chronicle/queue/impl/single/SingleChronicleQueueTest.java

```diff
@@ -2476,7 +2481,8 @@ public void testTailerWhenCyclesWhereSkippedOnWrite() {
         }
     }
 
-    private void doSomething(@NotNull ThreadLocal<ExcerptAppender> tla, @NotNull ThreadLocal<ExcerptTailer> tlt, String text) {
+    private void doSomething
```

Comment:
can't see the benefit of auto-reformatting in this PR - it is occluding the real changes

Reply:
It accidentally happened. Sorry.